shell/completion.zsh

```diff
@@ -10,7 +10,7 @@
 # - $FZF_COMPLETION_OPTS    (default: empty)
 
 [[ -o interactive ]] || return 0
-
+[[ "${FZF_COMPLETION_TRIGGER-x}" != "" ]] || return 0
```

Comment:
Shouldn't do this because an empty trigger is allowed in zsh.

Related: https://github.com/junegunn/fzf/wiki/Configuring-fuzzy-completion#dedicated-completion-key

```suggestion
```

Reply:
Well, I wasn't too happy with it anyway, since it looks a bit random to just hook on one variable per disable-able feature.

But if you think that this whole thing is a good addition, then maybe:
* Use a random non-empty string value, something like `DISABLE` etc.
* Replace the whole thing with a single `$DISABLE` that could be set to a value like `ctrl_t:alt_r:completion`.
* Or anything similar.

I'll be happy with whatever allows me to disable these without copying the file.